include/web_video_server/web_video_server.h

```diff
@@ -2,7 +2,7 @@
 #define WEB_VIDEO_SERVER_H_
 
 #include <rclcpp/rclcpp.hpp>
-#include <cv_bridge/cv_bridge.h>
+#include <cv_bridge/cv_bridge.hpp>
```

Comment:
In my case, using humble the cv_bridge.hpp is not found by the compilator

```suggestion
#include <cv_bridge/cv_bridge.h>
```

Reply:
> In my case, using humble the cv_bridge.hpp is not found by the compilator

The cv_bridge.h header seems to be deprecated:
`#warning This header is obsolete, please include cv_bridge/cv_bridge.hpp instead`

Since we are compiling using -werror, this breaks compilation for us.
Is there a compile time switch that we can use to check for the ros2 version?

Reply:
Yay, I just notice the root cause of the problem. It is because the humble branch of cv_bridge has not been updated with the new header file. Still, we can fix it here at compile time by detecting ROS distro at compile time by doing something like that.

```suggestion
#ifdef USE_CV_BRIDGE_H
#include <cv_bridge/cv_bridge.h>
#else
#include <cv_bridge/cv_bridge.hpp>
#endif
```

Reply:
This has been fixed in #149